java/gtnh/egg-greg-tech--new-horizons.json

```
@@ -51,6 +51,26 @@
             "user_editable": true,
             "rules": "required|boolean",
             "field_type": "text"
+        },
+        {
+            "name": "Delete During Install \/ Reinstall",
+            "description": "These files will be deleted during install \/ reinstall",
+            "env_variable": "DELUPDATE",
+            "default_value": "config libraries mods resources scripts lwjgl3ify-forgePatches.jar java9args.txt startserver-java9.bat startserver-java9.sh",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "string",
+            "field_type": "text"
+        },
+        {
+            "name": "Do not delete these during Install \/ Reinstall",
+            "description": "Use this to prevent the script from deleting them during install \/ reinstall. Useful to prevent items inside folders which are deleted from being deleted\r\nExample: config\/JourneyMapServer",
+            "env_variable": "DELUPDATEEXLUDE",
```

Some wording changes and change variable name
```suggestion
            "name": "Keep During Install \/ Reinstall",
            "description": "Use this to prevent the script from deleting them during install \/ reinstall. Useful to retain items inside folders which are deleted\r\nExample: config\/JourneyMapServer",
            "env_variable": "KEEP_ON_INSTALL",
```